proposals/NNNN-New-vehicle-data-HandsOffSteering.md

```diff
@@ -0,0 +1,112 @@
+# New vehicle data HandsOffSteering
+
+* Proposal: [SDL-NNNN](NNNN-New-vehicle-data-HandsOffSteering.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | Policy Server | SHAID | iOS | Java Suite | RPC]
+
+## Introduction
+
+This proposal is to add new vehicle data `HandsOffSteering` to know if driver has hands on steering or not.
```

Comment:
```suggestion
This proposal is to add new vehicle data `HandsOffSteering` to know whether or not a driver has his or her hands on the steering wheel.
```

Reply:
Please also make this change in the PR description.

Reply:
Done